src/views/sign-in.ejs

```diff
@@ -27,9 +27,9 @@
                 <a title="Forcer le niveau eIDAS renvoyé par le Fournisseur d'Identité ne fonctionnera que si le Fournisseur de Service a précisé le paramètre &quot;acr_values&quot; dans l'appel à FranceConnect. Le cas échéant, le niveau faible (eidas1) sera renvoyé par FranceConnect">(info)</a>
             </label>
             <select class="form-control" required name="acr" placeholder="Niveau eIDAS (à renvoyer à FC)*">
-                <option value="eidas1" selected>Faible (eidas1)</option>
-                <option value="eidas2">Substantiel (eidas2)</option>
-                <option value="eidas3">Élevé (eidas3)</option>
+                <option value="eidas1" <%= !acr || acr && acr == 'eidas1' ? 'selected' : '' %>>Faible (eidas1)</option>
```

Comment:
We can be confident about `acr`being defined with the `getEidasLevel()` function, so we can skip the boolean test.

Moreover, default level is `eidas3`, not `eidas1`